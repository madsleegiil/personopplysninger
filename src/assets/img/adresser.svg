<?xml version="1.0" encoding="UTF-8"?>
<svg width="62px" height="62px" viewBox="0 0 62 62" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 53.2 (72643) - https://sketchapp.com -->
    <title>book-contact-2 Copy 2</title>
    <desc>Created with Sketch.</desc>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="book-contact-2-Copy-2" fill-rule="nonzero">
            <path d="M48.8972041,17.4675245 C49.1191722,17.6871982 49.4151295,17.8051282 49.7157112,17.8051282 C50.3515573,17.8051282 50.8717949,17.2871608 50.8717949,16.6489511 L50.8717949,6.24335665 C50.8717949,5.60514685 50.3538694,5.08717949 49.7157112,5.08717949 L39.310959,5.08717949 C38.8439012,5.08717949 38.4207748,5.36928672 38.2427378,5.80169698 C38.0623888,6.23179487 38.161812,6.72895104 38.4924518,7.05961771 L48.8972041,17.4675245 Z" id="Fill/home-1" fill="#78706A"></path>
            <polygon id="Path" fill="#C86151" points="5.08717949 23.5898947 5.08717949 61.0461538 22.8923077 61.0461538 22.8923077 38.1538462 38.1538462 38.1538462 38.1538462 61.0461538 55.9589744 61.0461538 55.9589744 23.5898947 30.5230769 -5.32907052e-14"></polygon>
            <path d="M60.6966193,26.1090918 C60.6864456,26.1023742 60.6813587,26.0844605 60.6711852,26.0732646 L31.4219038,0.322445044 C30.9361114,-0.107481681 30.1095013,-0.107481681 29.6237089,0.322445044 L0.374427556,26.0732646 C-0.0859306977,26.47856 -0.152059508,27.1458422 0.346449983,27.620553 C0.923805362,28.1288517 1.69954716,28.054958 2.15481858,27.6630978 C2.15990543,27.6608585 2.16753567,27.6586194 2.1726225,27.6563802 L30.524078,2.69823804 L58.8729903,27.6586194 C58.878077,27.6608585 58.8831638,27.6630978 58.8882507,27.665337 C59.3714997,28.0795892 60.1726756,28.0795892 60.6533811,27.665337 C60.658468,27.6630978 60.6660983,27.6608585 60.6711852,27.6586194 C61.1340869,27.2510846 61.1976721,26.5748457 60.6966193,26.1090918 Z" id="Path" fill="#59514B"></path>
        </g>
    </g>
</svg>