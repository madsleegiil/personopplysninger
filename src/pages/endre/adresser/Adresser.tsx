import React from "react";
import RedirectKnapp from "components/knapper/Redirect";
import { withRouter, RouteComponentProps } from "react-router";
import Brodsmulesti from "../../forside/sections/2-brodsmulesti/Brodsmulesti";
import MedPersonInfo from "../../../providers/personinfo/PersinInfo";
import VisAdresser from "../../forside/sections/4-personinfo/3-adresser/Adresser";

interface Routes {
  tjeneste?: string;
}

const EndreAdresser = (props: RouteComponentProps<Routes>) => (
  <MedPersonInfo>
    {({ adresser }) => (
      <div className="endreOpplysninger__container">
        <Brodsmulesti hierarki={[{ title: "side.endre.adresser" }]} />
        {adresser && <VisAdresser adresser={adresser} />}
        {props.match.params.tjeneste && (
          <RedirectKnapp tjeneste={props.match.params.tjeneste} />
        )}
      </div>
    )}
  </MedPersonInfo>
);

export default withRouter(EndreAdresser);
