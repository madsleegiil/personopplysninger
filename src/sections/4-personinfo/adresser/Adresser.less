.addresse {
  &__box {
  }
  &__tittel {
    padding: 0 0 0.5rem 0;
  }
  &__lesmer {
    padding: 0;
  }
  &__linje {
    width: 100%;
  }
}
