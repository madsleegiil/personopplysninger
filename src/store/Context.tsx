import React, { createContext, useContext, useReducer, Dispatch } from "react";
import { Action, Locale, reducer, Store } from "./Store";
import { FetchNameInfo } from "./providers/Auth";
import { FetchFeatureToggles } from "./providers/FeatureToggles";
import { FetchDsopInfo } from "../pages/digital-samhandling-offentlig-privat/DsopFetch";
import { FetchInstInfo } from "../pages/institusjonsopphold/InstFetch";
import { FetchPersonInfo } from "./providers/PersonInfo";
import { FetchKontaktInfo } from "../pages/forside/sections/4-personinfo/2-kontaktinfo/subsections/kontakt-og-reservasjonsregisteret/DKIF-Fetch";
import { Fetchskattetrekksmeldinger } from "../pages/skattetrekksmelding/SkattFetch";
import { FetchMedlInfo } from "../pages/medlemskap-i-folketrygden/MedlFetch";

interface Props {
  children: JSX.Element | JSX.Element[];
}

export const StoreContext = createContext({} as [Store, Dispatch<Action>]);
export const StoreProvider = (props: Props) => {
  const { children } = props;

  const initialLocale = (window.location.pathname.includes("/en")
    ? "en"
    : "nb") as Locale;

  const initialState = {
    locale: initialLocale,
    nameInfo: { status: "LOADING" } as FetchNameInfo,
    featureToggles: {
      status: "LOADING",
      data: {
        "personopplysninger.pdl": false,
        "personopplysninger.dsop": false,
        "personopplysninger.inst": false,
        "personopplysninger.skatt": false,
        "personopplysninger.medl": false,
        "personopplysninger.fullmakt": false,
        "personopplysninger.tilrettelegging": false,
        "pdl-fullmakt": false,
      },
    } as FetchFeatureToggles,
    dsopInfo: { status: "LOADING" } as FetchDsopInfo,
    instInfo: { status: "LOADING" } as FetchInstInfo,
    personInfo: { status: "LOADING" } as FetchPersonInfo,
    kontaktInfo: { status: "LOADING" } as FetchKontaktInfo,
    skattetrekksmeldinger: { status: "LOADING" } as Fetchskattetrekksmeldinger,
    medlInfo: { status: "LOADING" } as FetchMedlInfo,
  };

  return (
    <StoreContext.Provider value={useReducer(reducer, initialState)}>
      {children}
    </StoreContext.Provider>
  );
};
export const useStore = () => useContext(StoreContext);
