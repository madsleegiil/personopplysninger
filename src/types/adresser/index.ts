import { Boadresse } from "./boadresse";
import { Postadresse } from "./postadresse";
import { Tilleggsadresse } from "./tilleggsadresse";
import { UtenlandskAdresse } from "./utenlandskadresse";
import { Kontaktadresse } from "./kontaktadresse";

export interface GeografiskTilknytning {
  bydel?: string;
  datoFraOgMed?: string;
  kommune?: string;
  enhet?: string;
  land?: string;
}

export interface Adresser {
  boadresse?: Boadresse;
  geografiskTilknytning?: GeografiskTilknytning;
  postadresse?: Postadresse;
  prioritertAdresse?: string;
  kontaktadresse: Kontaktadresse;
  datakilder?: {}[];
}
