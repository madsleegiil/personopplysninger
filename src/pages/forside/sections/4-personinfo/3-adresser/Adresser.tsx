import React from "react";
import { Adresser as IAdresser } from "types/adresser";
import Box from "components/box/Box";
import adresseIkon from "assets/img/Adresse.svg";
import driftsmeldinger from "driftsmeldinger";
import { AlertStripeAdvarsel } from "nav-frontend-alertstriper";
import Folkeregisteret from "./kontaktadresse/visning/Folkeregisteret";
import Kontaktadresser from "./kontaktadresse/visning/Kontaktadresser";

interface Props {
  adresser: IAdresser;
}

const Adresser = (props: Props) => {
  const { adresser } = props;
  const { kontaktadresser, bostedsadresse, deltBosted, oppholdsadresse } = adresser;

  return (
    <Box
      id="adresser"
      tittel="adresse.tittel"
      beskrivelse="adresse.beskrivelse"
      icon={adresseIkon}
      visAnkerlenke={true}
    >
      <div className="adresse__box">
        {driftsmeldinger.pdl && (
            <div style={{ padding: "1rem 0" }}>
              <AlertStripeAdvarsel>{driftsmeldinger.pdl}</AlertStripeAdvarsel>
            </div>
        )}

        <Folkeregisteret bostedsadresse={bostedsadresse} deltBosted={deltBosted} oppholdsadresse={oppholdsadresse} />
        <Kontaktadresser kontaktadresser={kontaktadresser}/>
      </div>
    </Box>
  );
};

export default Adresser;
