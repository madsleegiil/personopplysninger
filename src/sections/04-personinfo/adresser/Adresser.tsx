import React from "react";
import PropTypes from "prop-types";
import { InjectedIntlProps, injectIntl, FormattedMessage } from "react-intl";
import { Undertekst } from "nav-frontend-typografi";
import BoAdresse from "../../../components/adresse/BoAdresse";
import PostAdresse from "../../../components/adresse/PostAdresse";
import UtenlandskAdresse from "../../../components/adresse/UtenlandskAdresse";
import MidlertidigAdresse from "../../../components/adresse/MidlertidigAdresse";
import { Adresser } from "../../../types/adresser";
import Box from "../../../components/box/Box";
import hus from "../../../assets/img/hus.svg";

interface Props {
  adresser: Adresser;
}

const AdresseContainer = (props: Props & InjectedIntlProps) => {
  const { adresser, intl } = props;
  return (
    <Box
      id="adresse"
      header={intl.formatMessage({ id: "adresse.adresser" })}
      icon={hus}
      infoType="adresse"
    >
      <div>
        {adresser.boadresse && <BoAdresse boadresse={adresser.boadresse} />}
        {adresser.postadresse && (
          <PostAdresse postadresse={adresser.postadresse} />
        )}
        {adresser.tilleggsadresse && (
          <MidlertidigAdresse tilleggsadresse={adresser.tilleggsadresse} />
        )}
        {adresser.utenlandskAdresse && (
          <UtenlandskAdresse utenlandskadresse={adresser.utenlandskAdresse} />
        )}
      </div>
      <div className="box-footer">
        <Undertekst>
          <FormattedMessage id="adresse.source" />
        </Undertekst>
      </div>
    </Box>
  );
};

AdresseContainer.propTypes = {
  adresser: PropTypes.shape({}).isRequired
};

export default injectIntl(AdresseContainer);
