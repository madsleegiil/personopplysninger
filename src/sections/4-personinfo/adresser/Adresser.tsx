import React from "react";
import PropTypes from "prop-types";
import { InjectedIntlProps, injectIntl } from "react-intl";
import BoAdresse from "./varianter/BoAdresse";
import PostAdresse from "./varianter/PostAdresse";
import UtenlandskAdresse from "./varianter/UtenlandskAdresse";
import MidlertidigAdresse from "./varianter/MidlertidigAdresse";
import { Adresser } from "../../../types/adresser";
import Box from "../../../components/box/Box";
import hus from "../../../assets/img/hus.svg";
import LeggTilAdresse from "./LeggTilAdresse";

interface Props {
  adresser: Adresser;
}

const AdresseContainer = (props: Props & InjectedIntlProps) => {
  const { adresser, intl } = props;
  return (
    <Box
      id="adresse"
      tittel="adresse.tittel"
      beskrivelse="adresse.beskrivelse"
      icon={hus}
    >
      <>
        {adresser.boadresse && <BoAdresse boadresse={adresser.boadresse} />}
        {adresser.postadresse && (
          <PostAdresse postadresse={adresser.postadresse} />
        )}
        {adresser.tilleggsadresse && (
          <MidlertidigAdresse tilleggsadresse={adresser.tilleggsadresse} />
        )}
        {adresser.utenlandskAdresse && (
          <UtenlandskAdresse utenlandskadresse={adresser.utenlandskAdresse} />
        )}
        {!adresser.tilleggsadresse && !adresser.utenlandskAdresse && (
          <LeggTilAdresse />
        )}
      </>
    </Box>
  );
};

AdresseContainer.propTypes = {
  adresser: PropTypes.shape({}).isRequired
};

export default injectIntl(AdresseContainer);
