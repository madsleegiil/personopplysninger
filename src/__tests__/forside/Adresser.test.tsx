import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import { IntlProvider } from "react-intl";
import nbMessages from "text/nb";
import personInfo from "../../clients/apiMock/app/fetch/person-info.json";
import Adresser from "../../pages/forside/sections/4-personinfo/3-adresser/Adresser";
import { Adresser as AdresserType } from "types/adresser";
import { StoreProvider } from "../../store/Context";

jest.mock("react-modal");
jest.mock("nav-frontend-js-utils", () => ({
  ...jest.requireActual("nav-frontend-js-utils"),
  guid: () => "Mocked UUID",
}));

describe("Adresser", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <StoreProvider>
          <IntlProvider locale={"nb"} messages={nbMessages}>
            <Adresser
              adresser={personInfo.adresser as unknown as AdresserType}
            />
          </IntlProvider>
        </StoreProvider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
