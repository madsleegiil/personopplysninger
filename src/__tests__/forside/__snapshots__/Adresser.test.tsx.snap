// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Adresser renders correctly 1`] = `
<DocumentFragment>
  <div
    class="box__wrapper"
    id="adresser"
  >
    <div
      class="navds-guide-panel navds-guide-panel--poster"
    >
      <div
        class="navds-guide"
      >
        <div
          class="navds-guide__illustration navds-guide__illustration--medium"
        >
          <img
            alt=""
            class="box__ikon"
            src="Adresse.svg"
          />
        </div>
      </div>
      <div
        class="navds-guide-panel__content"
      >
        <div
          class="box__container"
        >
          <div
            class="box__header"
          >
            <div
              class="box__title-container"
            >
              <div
                class="box__line"
              />
              <h2
                class="box__title navds-heading navds-heading--medium"
              >
                Adresser
              </h2>
              <button
                aria-label="Les mer om denne seksjonen."
                aria-pressed="false"
                class="infotekst__title-i-button"
              >
                <img
                  alt=""
                  class="infotekst__title-i-icon"
                  src="Informasjonstekst.svg"
                />
              </button>
              <div
                class="box__line"
              />
            </div>
            <a
              aria-label="Direktelenke til denne seksjonen"
              class="navds-link anchor-link"
              href="#adresser"
            >
              <img
                alt=""
                src="AnchorLink.svg"
              />
              Lenke hit
            </a>
          </div>
          <div
            class="box__content"
          >
            <div
              class="adresse__box"
            >
              <div>
                <div
                  class="underseksjon__header"
                >
                  <h3
                    class="navds-heading navds-heading--small"
                  >
                    Adresser som er registert i Folkeregisteret
                  </h3>
                </div>
                <div
                  class="adresse__box"
                >
                  <div
                    class="underseksjon__header"
                  >
                    <h4
                      class="navds-heading navds-heading--xsmall"
                    >
                      Bostedsadresse
                    </h4>
                  </div>
                  <div
                    class="adresse__linje"
                  >
                    <p
                      class="navds-body-short"
                    >
                      V/ Aremark Jr
                    </p>
                  </div>
                  <div
                    class="adresse__linje"
                  >
                    <p
                      class="navds-body-short"
                    >
                      JØRGENSENVEIEN 14 B
                    </p>
                  </div>
                  <div
                    class="adresse__linje"
                  >
                    <p
                      class="navds-body-short"
                    >
                      0194 OSLO
                    </p>
                  </div>
                  <div
                    class="navds-read-more navds-read-more--medium adresse__lesmer"
                  >
                    <button
                      aria-expanded="false"
                      class="navds-read-more__button navds-body-short"
                      type="button"
                    >
                      <svg
                        aria-hidden="true"
                        class="navds-read-more__expand-icon"
                        fill="none"
                        focusable="false"
                        height="1em"
                        role="img"
                        viewBox="0 0 24 24"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          clip-rule="evenodd"
                          d="M12 17.414 4.293 9.707l1.414-1.414L12 14.586l6.293-6.293 1.414 1.414L12 17.414Z"
                          fill="currentColor"
                          fill-rule="evenodd"
                        />
                      </svg>
                      <svg
                        aria-hidden="true"
                        class="navds-read-more__expand-icon navds-read-more__expand-icon--filled"
                        fill="none"
                        focusable="false"
                        height="1em"
                        role="img"
                        viewBox="0 0 24 24"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          clip-rule="evenodd"
                          d="m12 18.121-8.06-8.06 2.12-2.122L12 13.88l5.94-5.94 2.12 2.122L12 18.12Z"
                          fill="currentColor"
                          fill-rule="evenodd"
                        />
                      </svg>
                      <span>
                        Vis detaljer
                      </span>
                    </button>
                    <div
                      style="height: 0px; overflow: hidden;"
                    >
                      <div
                        aria-hidden="true"
                        style="display: none;"
                      >
                        <div
                          class="navds-read-more__content navds-body-long"
                        >
                          <dl
                            class="list address-columns"
                          >
                            <div
                              class="list-item"
                            >
                              <dt>
                                <span
                                  class="navds-label"
                                >
                                  Bolignummer
                                </span>
                              </dt>
                              <dd>
                                H0301
                              </dd>
                            </div>
                            <div
                              class="list-item"
                            >
                              <dt>
                                <span
                                  class="navds-label"
                                >
                                  Kommune
                                </span>
                              </dt>
                              <dd>
                                EVJE OG HORNNES
                              </dd>
                            </div>
                            <div
                              class="list-item"
                            >
                              <dt>
                                <span
                                  class="navds-label"
                                >
                                  Flyttedato
                                </span>
                              </dt>
                              <dd>
                                14.07.2020
                              </dd>
                            </div>
                          </dl>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    class="adresse__divider"
                  />
                </div>
                <div
                  class="adresse__box"
                >
                  <div
                    class="underseksjon__header"
                  >
                    <h4
                      class="navds-heading navds-heading--xsmall"
                    >
                      Delt bosted
                    </h4>
                  </div>
                  <div
                    class="adresse__linje"
                  >
                    <p
                      class="navds-body-short"
                    >
                      V/ Aremark Jr
                    </p>
                  </div>
                  <div
                    class="adresse__linje"
                  >
                    <p
                      class="navds-body-short"
                    >
                      JØRGENSENVEIEN 14 B
                    </p>
                  </div>
                  <div
                    class="adresse__linje"
                  >
                    <p
                      class="navds-body-short"
                    >
                      0194 OSLO
                    </p>
                  </div>
                  <div
                    class="navds-read-more navds-read-more--medium adresse__lesmer"
                  >
                    <button
                      aria-expanded="false"
                      class="navds-read-more__button navds-body-short"
                      type="button"
                    >
                      <svg
                        aria-hidden="true"
                        class="navds-read-more__expand-icon"
                        fill="none"
                        focusable="false"
                        height="1em"
                        role="img"
                        viewBox="0 0 24 24"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          clip-rule="evenodd"
                          d="M12 17.414 4.293 9.707l1.414-1.414L12 14.586l6.293-6.293 1.414 1.414L12 17.414Z"
                          fill="currentColor"
                          fill-rule="evenodd"
                        />
                      </svg>
                      <svg
                        aria-hidden="true"
                        class="navds-read-more__expand-icon navds-read-more__expand-icon--filled"
                        fill="none"
                        focusable="false"
                        height="1em"
                        role="img"
                        viewBox="0 0 24 24"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          clip-rule="evenodd"
                          d="m12 18.121-8.06-8.06 2.12-2.122L12 13.88l5.94-5.94 2.12 2.122L12 18.12Z"
                          fill="currentColor"
                          fill-rule="evenodd"
                        />
                      </svg>
                      <span>
                        Vis detaljer
                      </span>
                    </button>
                    <div
                      style="height: 0px; overflow: hidden;"
                    >
                      <div
                        aria-hidden="true"
                        style="display: none;"
                      >
                        <div
                          class="navds-read-more__content navds-body-long"
                        >
                          <dl
                            class="list address-columns"
                          >
                            <div
                              class="list-item"
                            >
                              <dt>
                                <span
                                  class="navds-label"
                                >
                                  Bolignummer
                                </span>
                              </dt>
                              <dd>
                                H0301
                              </dd>
                            </div>
                            <div
                              class="list-item"
                            >
                              <dt>
                                <span
                                  class="navds-label"
                                >
                                  Kommune
                                </span>
                              </dt>
                              <dd>
                                EVJE OG HORNNES
                              </dd>
                            </div>
                          </dl>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    class="adresse__divider"
                  />
                </div>
                <div
                  class="adresse__box"
                >
                  <div
                    class="underseksjon__header"
                  >
                    <h4
                      class="navds-heading navds-heading--xsmall"
                    >
                      Oppholdsadresse
                    </h4>
                  </div>
                  <div
                    class="adresse__linje"
                  >
                    <p
                      class="navds-body-short"
                    >
                      V/ Aremark Jr
                    </p>
                  </div>
                  <div
                    class="adresse__linje"
                  >
                    <p
                      class="navds-body-short"
                    >
                      JØRGENSENVEIEN 14 B
                    </p>
                  </div>
                  <div
                    class="adresse__linje"
                  >
                    <p
                      class="navds-body-short"
                    >
                      0194 OSLO
                    </p>
                  </div>
                  <div
                    class="adresse__divider"
                  />
                </div>
                <div
                  class="kilde__container"
                >
                  <p
                    class="navds-body-long"
                  >
                    <a
                      class="navds-link kilde__lenke lenke"
                      href="https://www.skatteetaten.no/person/folkeregister/flytte/"
                    >
                      <span
                        class="kilde__icon"
                      >
                        <img
                          alt=""
                          src="Link.svg"
                        />
                      </span>
                      Endre adresse i Folkeregisteret
                    </a>
                  </p>
                  <div
                    class="kilde__tekst"
                  >
                    <p
                      class="navds-detail"
                    >
                      Kilde: Folkeregisteret
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="underseksjon__header underseksjon__divider"
              >
                <h3
                  class="navds-heading navds-heading--small"
                >
                  Øvrige registrerte adresser
                </h3>
              </div>
              <div
                class="adresse__box"
              >
                <div
                  class="underseksjon__header"
                >
                  <h4
                    class="navds-heading navds-heading--xsmall"
                  >
                    Kontaktadresse registrert hos NAV
                  </h4>
                </div>
                <div
                  class="adresse__linje"
                >
                  <p
                    class="navds-body-short"
                  >
                    V/ Aremark Jr
                  </p>
                </div>
                <div
                  class="adresse__linje"
                >
                  <p
                    class="navds-body-short"
                  >
                    Postboks 10 Sørstrand
                  </p>
                </div>
                <div
                  class="adresse__linje"
                >
                  <p
                    class="navds-body-short"
                  >
                    0194 OSLO
                  </p>
                </div>
                <div
                  class="navds-read-more navds-read-more--medium adresse__lesmer"
                >
                  <button
                    aria-expanded="false"
                    class="navds-read-more__button navds-body-short"
                    type="button"
                  >
                    <svg
                      aria-hidden="true"
                      class="navds-read-more__expand-icon"
                      fill="none"
                      focusable="false"
                      height="1em"
                      role="img"
                      viewBox="0 0 24 24"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clip-rule="evenodd"
                        d="M12 17.414 4.293 9.707l1.414-1.414L12 14.586l6.293-6.293 1.414 1.414L12 17.414Z"
                        fill="currentColor"
                        fill-rule="evenodd"
                      />
                    </svg>
                    <svg
                      aria-hidden="true"
                      class="navds-read-more__expand-icon navds-read-more__expand-icon--filled"
                      fill="none"
                      focusable="false"
                      height="1em"
                      role="img"
                      viewBox="0 0 24 24"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clip-rule="evenodd"
                        d="m12 18.121-8.06-8.06 2.12-2.122L12 13.88l5.94-5.94 2.12 2.122L12 18.12Z"
                        fill="currentColor"
                        fill-rule="evenodd"
                      />
                    </svg>
                    <span>
                      Vis detaljer
                    </span>
                  </button>
                  <div
                    style="height: 0px; overflow: hidden;"
                  >
                    <div
                      aria-hidden="true"
                      style="display: none;"
                    >
                      <div
                        class="navds-read-more__content navds-body-long"
                      >
                        <dl
                          class="list address-columns"
                        >
                          <div
                            class="list-item"
                          >
                            <dt>
                              <span
                                class="navds-label"
                              >
                                Gyldig til og med
                              </span>
                            </dt>
                            <dd>
                              20.07.2020
                            </dd>
                          </div>
                        </dl>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="adresse__divider"
                />
              </div>
              <button
                class="adresse__slett-kontaktadresse knapp-med-ikon navds-button navds-button--tertiary navds-button--medium"
              >
                <span
                  aria-live="polite"
                  class="navds-label"
                >
                  <span
                    class="kilde__icon"
                  >
                    <img
                      alt="Ekstern lenke"
                      src="Slett.svg"
                    />
                  </span>
                  <p
                    class="navds-body-long"
                  >
                    Slett kontaktadresse
                  </p>
                </span>
              </button>
              <div
                class="kilde__container"
              >
                <div
                  class="kilde__tekst"
                >
                  <p
                    class="navds-detail"
                  >
                    Kilde: NAV
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
